code/modules/events/immovablerod.dm

```diff
@@ -49,7 +49,6 @@
 	throwforce = 100
 	density = 1
 	anchored = 1
-	locked_atoms = list()
```

Comment:
Doesn't need to be here. Did get a runtime from spawning an immovable rod. Since fixed